src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -22,6 +22,9 @@
     <Compile Remove="AvxIntrinsics.cs" />
 
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
```

Comment:
This isn't right. How did this get here?

Reply:
This reference is required for the netcoreapp build. as it was earlier just in netstandard block, it was giving me build error

Reply:
I've opened #1688 for this issue.

Reply:
so should I keep it for this PR ?